TKGame/Debug.cs

```diff
@@ -0,0 +1,157 @@
+﻿using System;
+using System.Collections.Generic;
+using FontStashSharp;
+using Microsoft.Xna.Framework;
+using System.IO;
+
+// Myra is a library that allows us to add GUI components.
+// https://github.com/rds1983/Myra
+using Myra.Graphics2D.UI;
+using Microsoft.Xna.Framework.Input;
+
+namespace TKGame
+{
+    public class GameDebug
+    {
+        public static bool DebugMode { private get; set; }
```

Comment:
You can add summary comments to variables and properties. Might be useful for others who may need to use your variables in the future.

Reply:
Also fair, I'll see what I can add that'll help and push it up